python/activator/middleware_interface.py

```diff
@@ -75,22 +79,37 @@ class MiddlewareInterface:
         appropriate for use in the Google Cloud environment; typically only
         change this when running local tests.
     """
+
     def __init__(self, central_butler: Butler, image_bucket: str, instrument: str,
                  butler: Butler,
-                 prefix: str = "gs://"):
+                 prefix: str = "s3://"):
```

Comment:
This change is no longer necessary after the last weekly, and I've removed S3 support from the Cloud service.